sdk.mjs

```diff
@@ -190,10 +174,7 @@ export const normalizeColor = (text) => {
  * @returns {Promise<ThirdPartyExtension[]>} Information about third party extensions
  */
 export const getThirdPartyExtensions = async (
-  readmePath = path.join(
-    getDirnameFromImportMeta(import.meta.url),
-    'README.md',
-  ),
+  readmePath = path.join(import.meta.dirname, 'README.md'),
```

Comment:
This change needs to be applied to the new function `getThirdPartyLibraries` too.